docs/platforms/java/common/performance/instrumentation/apollo.mdx

```diff
@@ -1,6 +1,7 @@
 ---
 title: Apollo Integration
 sidebar_order: 30
+sdk: sentry.java.apollo
```

Comment:
```suggestion
sdk: io.sentry:sentry-apollo
```

https://github.com/getsentry/sentry-java/blob/8fd337bbd12b54caec15e8ab3dc0f71f02b67d72/sentry-apollo/src/main/java/io/sentry/apollo/SentryApolloInterceptor.kt#L44

Reply:
true, these aren't SDKs - however the docs currently use this field to find correlating packages (of all kinds) in our release registry.

the format this requires to work is e.g. `sentry.java.apollo` because it looks up the corresponding "sdk" here: 

https://github.com/getsentry/sentry-release-registry/tree/a7868f59c0f1a059c0e167473338f9c07e987360/sdks

so your suggestion feels more correct but would require changes in the release registry. the same applies to the other packages updated in this PR